adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -853,6 +853,10 @@ def _wait_for_msg(self, timeout=0.1):
         # read message contents
         raw_msg = self._sock_exact_recv(sz)
         msg = raw_msg if self._use_binary_mode else str(raw_msg, "utf-8")
+        if self.logger:
```

Comment:
Check `if self.logger is not None:` instead, please.

Reply:
I updated this and other conditionals to use the is not None:  

I have been able to successfully exercise some of the standard use cases, but none of the error based use cases.